Sources/Internal/OperationsManager.swift

```diff
@@ -18,6 +18,18 @@ import Foundation
 class OperationsManager {
   private var grpcClient: GrpcClient
 
+  private let queryRefAccessQ = DispatchQueue(
```

Comment:
Consider renaming `queryRefAccessQ` to `queryRefAccessDispatchQueue` or `queryRefAccessDispatcher` or something like that. Personally, I found "Q" to be confusing. Same with `mutationRefAccessQ`.

Reply:
Changed the name to queryRefAccessQueue and mutationRefAccessQueue.